docs/officers/travel/data-model.rst

```diff
@@ -0,0 +1,67 @@
+:og:description: Apiary models a trip as a single event with many travelers. A trip assignment links each traveler to a trip.
+
+Data model
+==========
+
+.. vale write-good.Weasel = NO
+
+Apiary models a **trip** as a single event with many travelers. A **trip assignment** links each traveler to a trip.
+
+Trip
+----
+
+A **trip** has a single destination and many travelers.
+
+Trips have a few key attributes:
+
+.. vale write-good.E-Prime = NO
+
+- The **status** which determines whether a trip is visible to travelers
```

Comment:
```suggestion
- The **status** that determines whether a trip is visible to travelers
```

Reply:
I fiddled with this bullet point a bit before I opened the PR and I don't really like the current phrasing or your suggestion. This was my other thought, but it doesn't fit the structure of the other bullets.

```suggestion
- The **status** determines whether a trip is visible to travelers
```

Reply:
I think that's better.